### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by hm90121**
- multi upload command
- multi download command 
- add blobber command
- replace blobber command
- change in indexing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,9 +34,14 @@ zbox is a command line interface (CLI) tool to understand the capabilities of Z
 
     - [Uploading and Managing files](#uploading-and-managing-files)
       - [Upload](#upload)
+        - [Upload file with no encryption](#upload-file-with-no-encryption)
+        - [Upload file with encryption](#upload-file-with-encryption)
+        - [Upload file with web streaming](#upload-file-with-web-streaming)
+        - [Mutli Upload](#multi-upload)
```
Comment (boddumanohar) [inline README.md]: `Multi`

File: README.md [hunk 2]
```diff
@@ -33,9 +34,14 @@ zbox is a command line interface (CLI) tool to understand the capabilities of Z
...
       - [Stream](#stream)
       - [Feed](#feed)
       - [Download](#download)
+        - [Mutli Download](#multi-download)
```
Comment (boddumanohar) [inline README.md]: `Multi`

File: README.md [hunk 3]
```diff
@@ -433,6 +438,56 @@ To test functionality try uploading file to allocation. You should get the follo
 Upload failed. this options for this file is not permitted for this allocation: 
 file_option_not_permitted.
 ```
+##### Add Blobber
+
+Use `add_blobber` flag with [update allocation](#update-allocation) command to add blobber to allocation.
```
Comment (boddumanohar) [inline README.md]: also please also add: 
"the new blobber will be added as a parity blobber"

please adjust the grammar accordingly

File: README.md [hunk 4]
```diff
@@ -433,6 +438,56 @@ To test functionality try uploading file to allocation. You should get the follo
...
+
+```
+Allocation updated with txId : d853a82907453d37ed978b9fc1a55663be99bb351d18cca31068c0dc95566edd
+```
+
+**Note:** Files will automatically be uploaded,splitted and stored on added blobber. 
+**Note:** An allocation is already hosted on a set of blobbers. To find a blobber that is available to add you should exclude the current set of blobbers hosting your allocation by checking them via [Get Allocation Info](#get)command.
+
+##### Replace Blobber
+
```
Comment (boddumanohar) [inline README.md:468]: Please add: 

Sometimes, a blobber might be mallfunctioning or faulty or the blobber might be slow becuase its far from your geolocation, in such cases you might have to replace the blobber with a  new one

**Comment by hm90121** [comment]
@boddumanohar  here is the commit for the required changes
https://github.com/0chain/zboxcli/commit/3b2d1f5a388ca7076ac54841412218c1d7887afe
its not showing in this PR please review and merge as appropriate.

**Review by cnlangzi — APPROVED** [review]

**Review by Kishan-Dhakan — APPROVED** [review]

**Pull request merged**